pkg/spire/spire_test.go

```diff
@@ -625,79 +618,47 @@ func TestSpire_TaskRunResultsSignTamper(t *testing.T) {
 				}
 				return prs
 			},
-			success: false,
+			wantErr: true,
 		},
 	}
 
 	for _, tt := range testCases {
-		ctx := context.Background()
-		for _, tr := range testTaskRuns() {
-			results := genPr()
-			success := func() bool {
-				resp := &fakeworkloadapi.X509SVIDResponse{
+		t.Run(tt.desc, func(t *testing.T) {
+			ctx := context.Background()
+			for _, tr := range testTaskRuns() {
+				results := pr
+
+				wl.SetX509SVIDResponse(&fakeworkloadapi.X509SVIDResponse{
 					Bundle: ca.X509Bundle(),
-					SVIDs:  makeX509SVIDs(ca, spiffeid.RequireFromPath(td, getTaskrunPath(tr))),
-				}
-				wl.SetX509SVIDResponse(resp)
+					SVIDs:  x509svids(ca, spiffeid.RequireFromPath(td, taskrunPath(tr))),
+				})
 
-				sigResults, err := ec.Sign(ctx, results)
-				if err != nil {
-					return false
-				}
+				var err error
+				var sigResults []v1beta1.PipelineResourceResult
+				if sigResults, err = ec.Sign(ctx, results); err == nil {
```

Comment:
I don't think needing to split the `wantErr` check is a strong enough reason to disregard https://google.github.io/styleguide/go/decisions#indent-error-flow.

Reply:
Much cleaner this way!